app/ide-desktop/lib/client/electron-builder-config.ts

```diff
@@ -70,9 +73,19 @@ const config: Configuration = {
     },
     copyright: 'Copyright © 2022 ${author}.',
     artifactName: 'enso-${os}-${version}.${ext}',
+    // Define a new protocol to enable redirects back to the IDE from external sources. This allows
+    // us to send the user to the browser to authenticate, and then redirect back to the IDE. This
+    // also allows us to intercept users clicking on signup verification links in their emails, also
+    // redirecting them back to the IDE.
+    protocols: [
```

Comment:
Lack of super important information - how oes it work and why it is defined in electron builder. The answer to the second is that electron builder registers this in system, so the app is opened when clicked on a deep link even if it was not started before. These things should be explained in docs.